.travis.yml

```diff
@@ -1,7 +1,8 @@
 rvm:
-  - 1.9.3
   - 2.0.0
   - 2.1
+  - 2.2
+  - 2.3.0
```

Comment:
ruby-2.3 is not installed - installing.
Searching for binary rubies, this might take some time.
Requested binary installation but no rubies are available to download, consider skipping --binary flag.
Gemset '' does not exist, 'rvm ruby-2.3 do rvm gemset create ' first, or append '--create'.
The command "rvm use 2.3 --install --binary --fuzzy" failed and exited with 2 during .